docs/team.rst

```diff
@@ -0,0 +1,77 @@
+Team
+====
+
+Write the Docs is a community effort that is enabled by a wonderful team of folks.
+We are always looking for more people to help organize events,
+meetups,
+or just take a leadership role in the community.
+This page documents the status of different members of the community,
+and also explains some of the roles that people are filling.
+
+These roles help folks join and contribute to the Write the Docs community. 
+Some of them are already filled,
+but if you're interested in helping out with any of them,
+just drop us a line at conf@writethedocs.org,
+or send one of us a DM on Slack.
+
+Moderation Team
+---------------
+
+* Moderate the Forum and Slack
+* Serve as points of contact for any issues or questions
+* Remove content that goes against the :doc:`/code-of-conduct`
+
+Members
+```````
+
+* `Becky Yoose <https://twitter.com/yo_bj>`_
+* `Eric Holscher <https://twitter.com/ericholscher>`_
+* `Jennifer Rondeau <https://twitter.com/bradamante>`_
+* `Kelly O'Brien <https://twitter.com/OBrienEditorial>`_
+* `Mike Jang <https://twitter.com/TheMikeJang>`_
+* `Mikey Ariel <https://twitter.com/thatdocslady>`_
+* `Neal Kaplan <https://twitter.com/NealKaplan>`_
+* `Samuel Wright <https://twitter.com/plaindocs>`_
+
+Meetup Coordinator Team
+-----------------------
+
+* Serve as point of contact for people interested in :doc:`/meetups/index`
+* Update website with new meetups
+* Socialize information around meetups (new meetup topics, speaker ideas)
+* Maintain documentation on starting meetups
+
+Members
+```````
+
+* `Mike Jang <https://twitter.com/TheMikeJang>`_
```

Comment:
I've added your basic bio to the line for now -- something to think more about in the future.